packages/react/src/components/WizardInline/__snapshots__/WizardInline.story.storyshot

```diff
@@ -6,14 +6,15 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/
 >
   <div
     className="WizardInline__StyledWizardWrapper-sc-8orpb0-0 hExrwn"
+    data-testid="wizard-inline"
   >
     <div
       className="bx--modal-container"
       data-id="WizardInlineContainer"
     >
       <div
         className="iot--wizard-inline__header page-title-bar"
-        data-testid="page-title-bar"
```

Comment:
Not a breaking change. This was only added in a previous testId PR. https://github.com/carbon-design-system/carbon-addons-iot-react/blame/9f42ed19798f47c3b6079ff0e091f534546551b6/packages/react/src/components/WizardInline/__snapshots__/WizardInline.story.storyshot